broScripts/parse_pcap.py

```diff
@@ -82,8 +107,7 @@ def is_person_empty(field):
                     continue
                 if line[0] == '[' and line[-2] == ']': # Check that this line represents an email
                     try:
-                        line = line.replace("\\'", "\'")
-                        headers = eval(line)
+                        headers = parseLine(line)
```

Comment:
Since we're changing how we parse lines, and we just write a stringified version of `headers` to our output file, do we need to modify Mike's Inbox class at all? Or is the format still consistent with what he expects?